package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "test": "karma start",
     "test:watch": "karma start --no-single-run --auto-watch",
+    "test:mac": "karma start --browsers Chrome",
+    "test:mac:watch": "karma start --browsers Chrome --no-single-run --auto-watch",
```

Comment:
Yooo, you should totally do something like the following in the karma config so we don't have to have to ways to launch tests:

```
const isWin = /^win/.test(process.platform);
...
const browsers = isWin ? ['IE', 'Chrome'] : ['Chrome']
```